kolibri/content/api.py

```diff
@@ -102,42 +101,26 @@ class Meta:
         fields = ['parent', 'search', 'prerequisite_for', 'has_prerequisite', 'related',
                   'recommendations_for', 'next_steps', 'popular', 'resume', 'ids', 'content_id', 'channel_id', 'kind', 'by_role']
 
-    def filter_search(self, queryset, name, value):  # noqa: C901
+    def filter_search(self, queryset, name, value):
         """
         Implement various filtering strategies in order to get a wide range of search results.
         """
-        # load stopwords file
-        stopwords_path = os.path.join(os.path.dirname(__file__), 'constants', 'stopwords-all.json')
-        with open(stopwords_path, 'r') as f:
-            stopwords = json.load(f)
-        stopwords_set = set()
-        for (key, values) in iteritems(stopwords):
-            stopwords_set.update(values)
-
         # return the result of and-ing a list of queries
         def intersection(queries):
-            return reduce(lambda x, y: x & y, queries)
-
-        # return the result of or-ing a list of queries
-        def union(queries):
-            return reduce(lambda x, y: x | y, queries)
+            if queries:
+                return reduce(lambda x, y: x & y, queries)
+            return []
```

Comment:
maybe return `None` instead? seems weird to have the function return either a `QuerySet` or an empty array

Reply:
Alternatively, could return an empty queryset, but I think this is fine.